com.unity.render-pipelines.core/Runtime/Common/CoreUnsafeUtils.cs

```diff
@@ -150,6 +151,233 @@ public static void CopyTo<T>(this T[] list, void* dest, int count)
                 UnsafeUtility.WriteArrayElement<T>(dest, i, list[i]);
         }
 
+        private static void RadixCalculateRadixParams(int radixBits, out int bitStates)
```

Comment:
To some extend, each method "calculate" or "compute" something.
Plus here there is two time "Radix". And the order of words are not compliant with standard english. AFAIK we neither really do prefixing this way.
Proposition: `GetRadixParams` / `ComputeRadixParams` / `CalculateRadixParams`